src/Platform.h

```diff
@@ -105,6 +105,7 @@ class Platform {
 	void setExitEventFilter();
 	bool dirCreate(const std::string& path);
 	bool dirRemove(const std::string& path);
+	std::string dirGetLocal();
```

Comment:
I only see the definition for GNU/Linux: how about other platforms?

Reply:
I don't use other platforms and i don't know if they have this issue. :D

Reply:
Me neither :smile:  A glance at the Windows equivalent yielded that the behavior is similar to what you're proposing (relative to the binary instead of pwd.  I suggest not exposing this method here then, since other platform is unlikely to define it.

Reply:
Windows does do the same thing, which is why we set the "Start In" field in the shortcut. But even then, it would only be 2 of the platforms (since MacOS/BSD/*nix uses the Linux platform). I would be okay with keeping the declaration+definition in the cpp file. In reality, this possibly doesn't even need to be a function where its only being used in one place.

Reply:
I made the suggested changes and i removed the function completely. Though i think it looked better with a function :D